openjdk/excludes/ProblemList_openjdk8.txt

```diff
@@ -322,6 +322,9 @@ sun/misc/Version/Version.java https://bugs.openjdk.java.net/browse/JDK-8244548 g
 com/sun/tools/attach/StartManagementAgent.java	https://github.com/adoptium/aqa-tests/issues/115	generic-all
 tools/pack200/CommandLineTests.java	https://github.com/adoptium/aqa-tests/issues/1051	linux-s390x
 tools/pack200/Pack200Test.java https://github.com/adoptium/aqa-tests/issues/2657 generic-all
+sun/tools/jstatd/TestJstatdExternalRegistry.java  https://github.com/adoptium/aqa-tests/issues/3061 linux-all
+sun/tools/jstatd/TestJstatdPort.java  https://github.com/adoptium/aqa-tests/issues/3061 linux-all
+sun/tools/jstatd/TestJstatdPortAndServer.java  https://github.com/adoptium/aqa-tests/issues/3061 linux-all
```

Comment:
Please reference the actual issue `https://bugs.openjdk.java.net/browse/JDK-8081569`, not `https://github.com/adoptium/aqa-tests/issues/3061`. https://github.com/adoptium/aqa-tests/issues/3061 is a task for excluding the tests and it will be closed by this PR.